05-JWT-Basics/starter/errors/custom-error.js

```diff
@@ -1,8 +1,7 @@
 class CustomAPIError extends Error {
-  constructor(message, statusCode) {
-    super(message)
-    this.statusCode = statusCode
```

Comment:
Should this line still be in the function? You seem to have deleted it and removed the parameter